packages/v4/patternfly-docs/content/design-guidelines/styles/colors/colors.md

```diff
@@ -163,16 +160,16 @@ import './colors.css';
 <Grid hasGutter>
   <GridItem xl={6} lg={12} className="ws-colors-gridtext">
     <p>
-      Shadows provide specifics about depth, direction of movement, and surface edges. Depending on the surface’s elevation and relationships to other surfaces, the type of shadow will change.
+      Shadows provide specifics about depth, movement direction, and surface edges. Depending on the surface’s elevation and relationships to other surfaces, the type of shadow will change.
     </p>
     <ColorSwatch label="Large" color="--pf-global--BoxShadow--lg">
-      This variable is used to indicate a large shadow for many components, such as alerts and modals.
+      This variable displays a large shadow for many components, such as alerts and modals.
     </ColorSwatch>
     <ColorSwatch label="Medium" color="--pf-global--BoxShadow--md">
-      This variable is used to indicate a medium shadow for many components, such as accordion and dropdowns.
+      This variable displays a medium shadow for many components, such as accordions and dropdowns.
     </ColorSwatch>
     <ColorSwatch label="Small" color="--pf-global--BoxShadow--sm">
-      This variable is used to indicate a small shadow for the card component.
+      This variable displays a small shadow for the card component.
```

Comment:
We use small shadows for all kinds of things. I think better descriptions than what components use what shadows might be something more conceptual, like the size of the box or amount of depth or whatever the reason is behind a shadow choice.